### ISSUE TITLE
Spec of Runtime fees

### ISSUE THREAD

**Issue by maxzaver** [near/nearcore#3355]
Currently it is very difficult to understand from the Runtime code which operations correspond to which fees and how they are measured. This information is even harder to understand than the Runtime spec itself, because this information is implicit and scattered all over the code base. Contract runtime team would benefit greatly from having a spec that describes how certain fees are associated with certain runtime operations. It might even be considered blocking contract runtime team. @evgenykuzyakov estimated that it would take 1-2 days.

**Comment by evgenykuzyakov** [comment]
Part one is done here: https://github.com/nearprotocol/NEPs/pull/117

**Comment by evgenykuzyakov** [comment]
deprioritizing

**Comment by Rishabhraghwendra18** [comment]
Hi @evgenykuzyakov are you working on this issue ?

**Comment by bowenwang1996** [comment]
I believe @jakmeier is working on something related

**Comment by jakmeier** [comment]
Yes, while I am not necessarily working on a spec, I am in this general vicinity.
More specifically, I have been looking at the parameters of our gas cost model and how we have come up with the estimated values for those parameters. The goal of my current work is to find out where the model has deficiencies. For example, under-charging can be considered a vulnerability as it opens the door to DoS attacks.
But I can definitely also offer my help on this issue, as I already have my head in the relevant details.

**Comment by bowenwang1996** [comment]
@jakmeier I think this might be a natural outcome of your investigation into the cost model :)
